Source/Core/Core/PrimeHack/Mods/FpsControls.cpp

```diff
@@ -778,6 +779,9 @@ void FpsControls::run_mod_mp3(Game active_game, Region active_region) {
 
   // Nothing new here
   write32(0, angular_momentum + 0x18);
+
+  u32 current_ball_state = read32(ball_state);
+  swap_morph_profiles(current_ball_state);
```

Comment:
This should also be added to the other Wii games supported, right now it's not explicitly clear to the user this is only for MP3 and it doesn't need to be either.

Reply:
Will do.